Task1.2/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Task1._2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Input 9 digits");
+            var digitInput = Console.ReadLine();
+            int multiple = 10;
+            int sum = 0;
+            foreach (var digit in digitInput.ToCharArray())
+            {
+                sum += multiple-- * (int)Char.GetNumericValue(digit);
+            }
+            int checkDigit = 11 - sum % 11;
```

Comment:
1. In if else use brackets {}{} for code block, and at least new lines. Code style.
2. Be careful with a task : "Create an application that prompts the user for a string of 9 digit characters (these are the first nine digits of the ISBN), calculates the check digit, and outputs the **resulting ISBN**